tests/torch/test_onnx_export.py

```diff
@@ -97,3 +102,15 @@ def test_exported_version(tmp_path: str, save_format: str, ref_opset: int):
     model_proto = onnx.load_model(onnx_checkpoint_path)
     # pylint: disable=no-member
     assert model_proto.opset_import[0].version == ref_opset
+
+
+def test_can_export_single_batch_bn(tmp_path):
+    test_path = tmp_path.joinpath('test.onnx')
+    synthetic_model_desc = SingleLayerModelDesc(layer=nn.BatchNorm2d(4), input_sample_sizes=([1, 4, 1, 1]))
```

Comment:
Here's the test for a "hack" with export batch norm having single value per channel in training mode.